Doc/library/io.rst

```diff
@@ -1038,6 +1038,13 @@ Text I/O
       .. versionadded:: 3.7
 
 
+   Example usage::
+
+      # docoding newlines for universal newlines mode.
```

Comment:
```suggestion
      # decoding newlines for universal newlines mode
```

but I am not sure I understand the comment.  do you mean: decoding newlines *in* universal newline mode?

Reply:
yes

Reply:
ok then please change for → in in both places

Reply:
done